variables.tf

```diff
@@ -308,3 +308,225 @@ variable "check_ec2_imdsv2" {
   type        = bool
   default     = false
 }
+
+variable "check_access_keys_rotated" {
+  description = "Enable access-keys-rotated rule"
+  type        = bool
+  default     = true
+}
+
+variable "access_key_max_age" {
+  description = "Maximum number of days without rotation."
+  type        = number
+  default     = 90
+}
+
+variable "check_cmk_backing_key_rotated" {
+  description = "Enable cmk_backing_key_rotation_enabled rule"
+  type        = bool
+  default     = true
+}
+
+variable "check_backup_plan_min_frequency_and_min_retention" {
+  description = "Enable backup_plan_min_frequency_and_min_retention_check rule"
+  type        = bool
+  default     = false
+}
+
+variable "backup_frequency" {
+  description = "Numerical value for required backup frequency. Maximum of 24 for hours, 31 for days."
+  type        = number
+  default     = 1
+}
+
+variable "retention_days" {
+  description = "Required retention period in days."
+  type        = number
+  default     = 35
```

Comment:
Why 35?

Reply:
Also `retention_days` what? Retention of logs? Retention of backups? I imagine both of these have unique values.

Reply:
35 is the AWS default. Would you recommend a default of 90? I feel like a solid 3 months retention is on par. 
I have changed this to `backup_retention_days`

Reply:
I am good leaving it as the AWS default currently then if you are.

Reply:
this rule has been removed.